e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js

```diff
@@ -273,6 +275,11 @@ function disableBasicModelActions(modelId) {
   cy.wait("@getModelActions");
 }
 
+function visitModel(modelId) {
+  cy.visit(`/model/${modelId}`);
+  cy.wait(["@allPostRequests", "@allGetRequests"]);
```

Comment:
I think simply waiting for the `@dataset` should work here. It's the last request that contains the data we use to populate the table.